resources/views/dashboard.blade.php

```diff
@@ -36,6 +53,16 @@
                       <?php $balanceStyle = array(['class'=>'', 'operator' => '']); ?>
                       @endif
                           <tr>
+                              <td>
+                                  @foreach ($allSharedExpenses as $sharedExpense)
```

Comment:
Dan this needs to check if the person is the owner of the expense (ie: the person being owed, in this iteration it allows for both people who are owed and people who owe to settle the expense)